daemon/system-info.sh

```diff
@@ -224,8 +227,15 @@ else
     LCPU_COUNT="$(grep -c ^processor /proc/cpuinfo)"
   fi
 
-  # If we have GNU uname, we can use that to get CPU info (probably).
-  if uname --version 2> /dev/null | grep -qF 'GNU coreutils'; then
+  if [ "${KERNEL_NAME}" = Darwin ]; then
+    CPU_MODEL="$(sysctl -n machdep.cpu.brand_string)"
+    if [ "${ARCHITECTURE}" = "x86_64" ]; then
+      CPU_VENDOR="$(sysctl -n machdep.cpu.vendor)"
+    else
+      CPU_VENDOR="Apple"
+    fi
```

Comment:
```suggestion
    if [ "${ARCHITECTURE}" = "x86_64" ]; then
      CPU_VENDOR="$(sysctl -n machdep.cpu.vendor)"
    fi
```

Don't hardcode vendor for m1?

Reply:
Can we reliably get this info even in cases where getting the CPU frequency fails?

Reply:
What do you mean?. The current implementation uses `sysctl` utility to retrieve CPU info. If the kernel returns the value we need - we are good, if not - 🤷‍♂️.

Reply:
> Can we reliably get this info even in cases where getting the CPU frequency fails?

Ah, I see. Yes, our tests:
 - me checked on m1 pro.
 - @cpipilas checked on an intel mac.